### PR TITLE
Fix download entry shown on public share menu when not in mobile

### PR DESCRIPTION
**Pull request by danxuliu**
This fixes a regression introduced in #15199 due to a typo in the variable name.

I have also added messages to `assertFalse/assertTrue` in the related acceptance tests (as [thanks to them](https://drone.nextcloud.com/nextcloud/server/19021/69/4) is how I found the issue ;-) ) to make the failures more descriptive.

### CODE REVIEW

File: apps/files_sharing/css/public.scss [hunk 1]
```diff
@@ -195,7 +195,7 @@ thead {
 
 // hide the download entry on the menu
 // on public share when NOT on mobile
-@media only screen and (min-width: $mobile_breakpoint + 1) {
+@media only screen and (min-width: $breakpoint-mobile + 1) {
```
Comment (skjnldsv) [inline apps/files_sharing/css/public.scss]: :see_no_evil:

**Review by skjnldsv — APPROVED** [review]

File: apps/files_sharing/css/public.scss (hunk 1 above)
Comment (nachoparker) [reply apps/files_sharing/css/public.scss]: :see_no_evil:

**Review by nachoparker — APPROVED** [review]

**Comment by skjnldsv** [comment]
https://drone.nextcloud.com/nextcloud/server/19215/68/4 passed

**Comment by nachoparker** [comment]
I tested the change but must have forgotten to remove the cached css :|

**Pull request merged**